src/data/dialogmote/useMellomlagreReferat.ts

```diff
@@ -0,0 +1,19 @@
+import { useMutation, useQueryClient } from "react-query";
+import { ISDIALOGMOTE_ROOT } from "@/apiConstants";
+import { NewDialogmoteReferatDTO } from "@/data/dialogmote/types/dialogmoteReferatTypes";
+import { post } from "@/api/axios";
+import { dialogmoterQueryKeys } from "@/data/dialogmote/dialogmoteQueryHooks";
+
+export const useMellomlagreReferat = (fnr: string, dialogmoteUuid: string) => {
```

Comment:
Kan denne hooken ligge i hooks/dialogmote mappa i stedet for data/dialogmote, eller er det skille på de på noe vis som gjør at den passer bedre her?

Reply:
Tenker at de hooksene som ligger i data/dialogmote har med henting/sending av data til API å gjøre så sånn sett synes jeg den passer bedre her.